admin_solutions/authentication.adoc

```diff
@@ -5,10 +5,308 @@ toc::[]
+provider's identities and user objects.
+.. Specify the `*url*` to use to connect to a server that accepts credentials in Basic authentication headers.
+.. Optionally, change the `*ca*` to the certificate bundle to use in order to validate server certificates for the configured URL.
+.. Optionally, change the `*certFile*` to the client certificate to present when making requests to the configured URL.
+.. If `*certFile*` is specified, then you must change the `*keyFile*` to the key for the client certificate.
+. Save your changes and close the file.
+. Start the {product-title} API server, specifying the configuration file you just
+modified:
++
+----
+$ openshift start master --config=<path/to/modified/config>/master-config.yaml
+----
+
+Once configured, any user logging in to the {product-title} web console will be
+prompted to log in using their Basic authentication credentials.
+
+[[basic-troubleshooting]]
+=== Troubleshooting
+
+[WARNING]
+====
+*[NEEDINFO]*: Is this troubleshooting information? It was in introduction of the install guide authentication topic, but seems out of place there. What purpose does this serve? Does any of it belong in the overview here, in other words does any of this need to be considered before configuring authentication?
+====
+
+A `401` response indicates failed authentication.
+
+A non-`200` status, or the presence of a non-empty "error" key, indicates an
+error:
+
+----
+{"error":"Error message"}
+----
+
+A `200` status with a `sub` (subject) key indicates success:
+
+----
+{"sub":"userid"} <1>
+----
+<1> The subject must be unique to the authenticated user and must not be able to
+be modified.
+
+A successful response may optionally provide additional data, such as:
+
+* A display name using the `name` key. For example:
++
+----
+{"sub":"userid", "name": "User Name", ...}
+----
++
+* An email address using the `email` key. For example:
++
+----
+{"sub":"userid", "email":"user@example.com", ...}
+----
++
+* A preferred user name using the `preferred_username` key. This is useful when
+the unique, unchangeable subject is a database key or UID, and a more
+human-readable name exists. This is used as a hint when provisioning the
+OpenShift user for the authenticated identity. For example:
++
+----
+{"sub":"014fbff9a07c", "preferred_username":"bob", ...}
+----
+
+[[create-users-basic-auth]]
+=== Creating Users with Basic Authentication
+
+You do not create users in {product-title} when integrating with an external
+authentication provider, such as, in this case, Basic authentication. The Basic
+authentication server is the system of record, meaning that users are defined
+there, and any user with a valid user name for the configured authentication
+server can log in.
+
+To add a user to {product-title}, the user must exist on the Basic
+authentication server, and if required you must create a new Basic
+authentication account for the user.
```

Comment:
a "basic authentication account" is not a thing... "Basic authentication" refers to the way the credentials are sent from OpenShift to the remote auth server, nothing more